src/main/webapp/app/course/plagiarism-cases/instructor-view/plagiarism-cases-instructor-view.component.ts

```diff
@@ -131,20 +141,45 @@ export class PlagiarismCasesInstructorViewComponent implements OnInit {
     }
 
     /**
-     * export the plagiarism cases in CSV format
+     * set placeholder for undefined values and sanitize the operators away
+     * @param value
+     * @private
+     */
+    private sanitizeCSVField(value: any): string {
+        if (value === null || value === undefined) {
+            return '-';
+        }
+        return String(value).replace(/;/g, '";"');
+    }
+
+    /**
+     * export the cases in CSV format
      */
     exportPlagiarismCases(): void {
-        const blobParts: string[] = ['Student Login,Exercise,Verdict, Verdict Date\n'];
+        const headers = ['Student Login', 'Matr. Nr.', 'Exercise', 'Verdict', 'Verdict Date', 'Verdict By'].map((header) => this.sanitizeCSVField(header));
```

Comment:
Remove `.map((header) => this.sanitizeCSVField(header));`, as it does not really make sense here.